pkg/gui/gui.go

```diff
@@ -327,15 +327,15 @@ func initialScreenMode(startArgs appTypes.StartArgs, config config.AppConfigurer
 	}
 }
 
-func initialContext(contextTree *context.ContextTree, startArgs appTypes.StartArgs) types.IListContext {
-	var initialContext types.IListContext = contextTree.Files
+func initialContext(contextTree *context.ContextTree, startArgs appTypes.StartArgs) types.Context {
+	var initialContext types.Context = contextTree.Files
 
 	if startArgs.FilterPath != "" {
 		initialContext = contextTree.LocalCommits
 	} else if startArgs.GitArg != appTypes.GitArgNone {
 		switch startArgs.GitArg {
 		case appTypes.GitArgStatus:
-			initialContext = contextTree.Files
+			initialContext = contextTree.Status
```

Comment:
I wasn't sure if this was intentional (maybe because our files panel is closer to what you get when you type "git status"?) Happy to drop this commit if it was.

Reply:
That's right: when you do a `git status` you expect to see changed files, so we're showing the files panel.

The 'status' context should really be called the 'repo' context

Reply:
OK, I dropped this commit.